src/digit/mod.rs

```diff
@@ -56,27 +57,311 @@ pub trait Digit: AsRef<str> {
 impl Digit for String {}
 impl Digit for str {}
 
+/// The multipliers of the persian number system, up to a billion.
+pub static MULTIPLIERS: phf::Map<&'static str, u32> = phf::phf_map! {
```

Comment:
I think nobody outside of this library needs these hashmaps, maybe we can make them private.